README.md

```diff
@@ -24,25 +24,21 @@ composer:
 curl -sS https://getcomposer.org/installer | php
 ```
 
-The Mailgun api client is not hard coupled to Guzzle or any other library that sends HTTP messages. It uses an abstraction 
-called HTTPlug. This will give you the flexibilty to choose what PSR-7 implementation and HTTP client to use. 
+The Mailgun api client is not hard coupled to Guzzle or any other library that sends
+HTTP messages. It uses the [PSR-18](https://www.php-fig.org/psr/psr-18/) client abstraction.
+This will give you the flexibilty to choose what PSR-7 implementation and HTTP client to use. 
 
 If you just want to get started quickly you should run the following command: 
 
 ```bash
-php composer.phar require mailgun/mailgun-php php-http/curl-client guzzlehttp/psr7
+composer require mailgun/mailgun-php kriswallsmith/buzz nyholm/psr7
 ```
 
-### Why requiring so many packages?
-
-Mailgun has a dependency on the virtual package
-[php-http/client-implementation](https://packagist.org/providers/php-http/client-implementation) which requires you to install **an** adapter, but we do not care which one. That is an implementation detail in your application. We also need **a** PSR-7 implementation and **a** message factory. 
-
-You do not have to use the `php-http/curl-client` if you do not want to. You may use the `php-http/guzzle6-adapter`. Read more about the virtual packages, why this is a good idea and about the flexibility it brings at the [HTTPlug docs](http://docs.php-http.org/en/latest/httplug/users.html).
-
 ## Usage
 
-You should always use Composer's autoloader in your application to automatically load the your dependencies. All examples below assumes you've already included this in your file:
+You should always use Composer's autoloader in your application to automatically load
+the your dependencies. All examples below assumes you've already included this in your
```

Comment:
Maybe removing `the` or `your`, but keeping both seems wrong to me.

Also, maybe adding `the` next to `All`?

Finally, `examples` is plural so `assumes` should be `assume`

```suggestion
your dependencies. All the examples below assume you've already included this in your
```

Reply:
Thank you. 

Also, cool. I never seen this "suggestion" feature before